source/extensions/common/wasm/wasm.cc

```diff
@@ -4,6 +4,7 @@
 
 #include <memory>
 #include <string>
+#include <vector>
```

Comment:
Why is this needed?

Reply:
Removed.  Left over from some merge/patch I presume.